pkgs/development/interpreters/ruby/build-ruby-gem/gem-post-build.rb

```diff
@@ -64,8 +64,7 @@
 # this file is here to facilitate running it.
 #
 
-gem_path = ENV["GEM_PATH"]
-ENV["GEM_PATH"] = "\#{gem_path}\#{":" unless gem_path.nil? || gem_path.empty?}#{(gem_path+[gem_home]).join(":")}"
+Gem.use_paths "#{gem_home}", #{gem_path.to_s}
 
 require 'rubygems'
```

Comment:
Ah, yes... I forget if there are still circumstances in which `rubygems` isn't loaded...

@zimbatm How do you suppose this should be changed? Should we just move the `require` above `Gem.use_paths` to be safe?